esmvaltool/recipes/ipccwg1ar5ch9/recipe_flato13ipcc_figures_926_927.yml

```diff
@@ -0,0 +1,186 @@
+# ESMValTool
+# recipe_flato13ipcc_figures_926_927.yml
+---
+documentation:
+  title: IPCC AR5, chap. 9 diagnostics for 9.26 and 9.27
+
+  description: |
+    Reproducing selected figures from IPCC AR5, chap. 9 (Flato et al., 2013)
+    9.26, 9.27.
+    Data sets taken out were part of the original publication but are
+    currently not available on ESGF.
+
+  authors:
+    - bock_lisa
+    - gier_bettina
+    - lauer_axel
+    - schlund_manuel
+    - senftleben_daniel
+    - zimmermann_klaus
+
+  maintainer:
+    - lauer_axel
+
+  references:
+    - flato13ipcc
+
+  projects:
+    - embrace
+    - esmval
+    - crescendo
+
+
+preprocessors:
+
+  land_fraction_weighting:
+    weighting_landsea_fraction: &weighting_options
+      area_type: land
+      exclude: [
+        # 'bcc-csm1-1-m',
+        'GCP2018',
+        # 'inmcm4',
+        'JMA-TRANSCOM',
+      ]
+
+  sea_fraction_weighting:
+    weighting_landsea_fraction:
+      <<: *weighting_options
+      area_type: sea
+    regrid:
+      target_grid: 2x2
+      scheme: linear
+
+
+diagnostics:
+
+  # **********************************************************************
+  # Flato et al. (2013) - IPCC AR5, chap. 9
+  # similar to fig. 9.26
+  # **********************************************************************
+  # Ensemble-mean global ocean carbon uptake (top) and global land
+  # carbon uptake (bottom) in the CMIP5 ESMs for the historical period
+  # 1901-2005.
+  # **********************************************************************
+
+  fig09-26top:
+    title: Fig. 9.26a (carbon uptake)
+    variables:
+      fgco2: &fgco2_settings
+        preprocessor: sea_fraction_weighting
+        project: CMIP5
+        mip: Omon
+        exp: historical
+        ensemble: r1i1p1
+        start_year: 1901
+        end_year: 2005
+        reference_dataset: JMA-TRANSCOM
+        plot_units: PgC y-1
+        additional_datasets:
+          # - {dataset: JMA-TRANSCOM, project: OBS, type: reanaly, version: 2018, tier: 3, start_year: 1986, end_year: 2005}
+          # - {dataset: CanESM2}
+          # - {dataset: CESM1-BGC}
+          # - {dataset: GFDL-ESM2G}
+          # - {dataset: GFDL-ESM2M}
+          # - {dataset: HadGEM2-CC}
+          # - {dataset: HadGEM2-ES}
+          # - {dataset: inmcm4}
+          # - {dataset: IPSL-CM5A-LR}
+          # - {dataset: IPSL-CM5A-MR}
+          # - {dataset: IPSL-CM5B-LR}
+          # - {dataset: MIROC-ESM}
+          # - {dataset: MIROC-ESM-CHEM}
+          - {dataset: MPI-ESM-LR}
+          - {dataset: MPI-ESM-MR}
+          # - {dataset: NorESM1-ME}
```

Comment:
Why are all these datasets commented out? With the current datasets, the diagnostic makes rather little sense.

Reply:
See other answer to "Specific comments", I added some more, but several are not available